### PR TITLE
MSP: displayport info

### PR DESCRIPTION
**Pull request by yaapu**

### CODE REVIEW

File: common/source/docs/common-msp-osd-overview.rst [hunk 1]
```diff
@@ -60,18 +71,18 @@ Panel Items Currently Supported
...
 
 OSD Panel Item Configuration
-============================
+----------------------------
 
 Each OSD panel item uses a set of three variables to be set: 
 
-- ``OSDn_ITEM_ENABLE`` - activates the respective item when set to 1.
-- ``OSDn_ITEM_X`` and ``OSDn_ITEM_Y`` set the horizontal and vertical position of the item, starting with ``X = 0`` and ``Y = 0`` in the upper left corner of your screen. 
+- ``OSDn_<ITEM>_ENA`` - activates the respective item when set to 1.
```
Comment (Hwurzburg) [inline common/source/docs/common-msp-osd-overview.rst:78]: OSDn<ITEM>_EN

also should give an example :ref:\`OSD2_BAT_BOLT_EN<OSD2_BAT_VOLT_EN>\`

**Comment by Hwurzburg** [comment]
@yaapu if okay with you, I am going to break this into two PRs..one applicable now with the format,etc stuff and one that adds the stuff to upcoming features ....will put you as co-author for commit credit....

**Comment by yaapu** [comment]
Sure, no worries. I think the table and comments are the most needed updates

**Comment by Hwurzburg** [comment]
has been added...closing

**Pull request closed without merging**